worker/jobTypes/githubJob.js

```diff
@@ -268,6 +338,40 @@ class GitHubJobClass {
                 'ERROR: makefile does not exist in /makefiles directory on meta branch.'
             );
         }
+        //set up env vars for all next-gen jobs
+        if(this.buildNextGen()){
+          const pathPrefix = await this.writeEnvFile(isProdDeployJob)
```

Comment:
We have an opportunity here -- to split out the writeEnvFile into a Gatsby adapter. That way all things Gatsby-related will be in one module. Then we know exactly where to go when things related to Gatsby change, *and* we can easily swap out Gatsby for another provider, maybe even at run time eventually.

Could we change writeEnvFile to constructing our GatsbyAdapter object -- by passing in the job -- and writing the file with an initEnv method? Then we can persist the prefix to the job object as well instead of returning it in this method.

When we eliminate the makefiles we will abstract these calls and allow the instantiated builder to manage the differences between the different build types.